Source/WebKit/DerivedSources.make

```diff
@@ -28,6 +28,8 @@ VPATH = \
     $(WebKit2)/GPUProcess/media \
     $(WebKit2)/GPUProcess/media/ios \
     $(WebKit2)/GPUProcess/webrtc \
+    $(WebKit2)/ModelProcess \
+    $(WebKit2)/ModelProcess/mac \
```

Comment:
There's nothing in this directory.  Could you remove this?  Also, let's put stuff in a directory named "cocoa" instead.